packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -1,115 +1,136 @@
 import React from 'react';
+import styled from 'styled-components';
+import { componentIds } from './componentIds';
 import Box from '~components/Box';
+import type { OptionsType } from '~components/Dropdown/useDropdown';
 import { useDropdown } from '~components/Dropdown/useDropdown';
-import { Text } from '~components/Typography';
-import { getPlatformType, makeAccessible } from '~utils';
-
-type ActionListItemProps = {
-  title: string;
-  value: string;
-  href?: string;
-  /**
-   * Internally passed from ActionList. No need to pass it explicitly
-   */
-  index?: number;
+import {
+  getComponentId,
+  isReactNative,
+  isValidAllowedChildren,
+  makeAccessible,
+  makeSize,
+} from '~utils';
+
+type ActionListProps = {
+  children: React.ReactNode[];
+  surfaceLevel?: 2 | 3;
 };
-const ActionListItem = (props: ActionListItemProps): JSX.Element => {
+
+const StyledActionList = styled(Box)<{ surfaceLevel: ActionListProps['surfaceLevel'] }>(
+  ({ theme, surfaceLevel = 2 }) => {
+    const offsetX = theme.shadows.offsetX.level[1];
+    const offsetY = theme.shadows.offsetY.level[2];
+    const blur = theme.shadows.blurRadius.level[2];
+    const shadowColor = theme.shadows.color.level[1];
+
+    const elevation200 = `${makeSize(offsetX)} ${makeSize(offsetY)} ${makeSize(
```

Comment:
In android we need additional logic to make them look similar to web, otherwise the shadows will look vastly different, check the CardSurface.native.tsx file to see how I did it.

Reply:
done in upcoming commit